numpy/lib/function_base.py

```diff
@@ -907,8 +907,8 @@ def gradient(f, *varargs, **kwargs):
     Returns
     -------
     gradient : ndarray
```

Comment:
The return type should also be updated in this line, either to `list` alone or to `list of ndarray`. See e.g. the docstring of `np.nonzero`, which returns a tuple of ndarrays [here](https://github.com/numpy/numpy/blob/v1.9.1/numpy/core/fromnumeric.py#L1396).